### DISCUSSION TITLE
HMIP-RFUSB and Synology

### DISCUSSION THREAD

**Discussion by alpha0x1omega** [OpenCCU/OpenCCU#2169] [🇩🇪 Fragen&Antworten] [answered]
Hallo RaspberryMatic Forum,

leider bekome ich keine Funkverbindung zwischen HMIP-RFUSB  und HM Geräte hin.
Ich bin gewechselt von CCU2 auf Synology mit virtual manager und OVA Version.  In RaspberryMatic Hilfe werden steht der USB Stick, siehe Anhang 1. Auch kann ich die Geräte Anlern Funktion in RaspberryMatic starten.  Das HM Gerät setzte ich ich auch in den Anlern Modus. Aber es wird nicht erkannt.  Den USB stick habe ich mit dem passenden Kabel weit weg gelegt um keinen Störungen zu empfangen. Hat einer von Euch noch einen Tipp. Ich habe jetzt schon das zweite USB zusammengelötet. Was könnte ich noch prüfen.  Bin für jede Anregung dankbar. Was könnte ich noch prüfen? 

Mit freundlichen Grüßen
Robert Kraus 

```
homematicIP-RF (HmIP) Info
RF-Module/Firmware:	C (4.4.18)
Device-Node:	/dev/raw-uart (eQ-3 HmIP-RFUSB@usb-0000:00:18.0-1)
Address:	0x16AB51 (0xBD5CF3)
SGTIN:	3014F711A000041F299ED052
Serial:	1F299ED052
HomeMatic-RF (HmRF) Info
RF-Module/Firmware:	HMIP-RFUSB (4.4.18)
Device-Node:	/dev/raw-uart (eQ-3 HmIP-RFUSB@usb-0000:00:18.0-1)
Address:	0xFF7DAE (0xFF7DAE)
Serial:	1F299ED052
```

**Comment by MichaelN0815** [comment, marked as answer]
Wie hoch ist der CarrierSense?

**Reply by alpha0x1omega** [reply]
Der CarrierSense ist 0%.

**Comment by stefanhintermayr** [comment]
Hast Du eine Lösung? Ich habe das gleiche Problem.

**Comment by alpha0x1omega** [comment]
Nachdem auch ein zweites HmIP-RFUSB nicht zusammen mit Raspberrymatic auf Virtual Machine Synology die Daten geliefert hat, habe ich mir bei ebay eine ccu3 ersteigert. Seitdem alles gut. Redundanz minimiert Ausfall Risiko. Elv hat auf Kulanz HMIP RFUSB zurück genommen Von meinem/meiner Galaxy gesendet
-------- Ursprüngliche Nachricht --------Von: stefanhintermayr ***@***.***> Datum: 06.03.23  19:27  (GMT+01:00) An: jens-maus/RaspberryMatic ***@***.***> Cc: alpha0x1omega ***@***.***>, Author ***@***.***> Betreff: Re: [jens-maus/RaspberryMatic] HMIP-RFUSB and Synology (Discussion #2169) 
Hast Du eine Lösung? Ich habe das gleiche Problem.

**Comment by Kaiiide** [comment]
Ich habe das gleiche Problem. Bei mir kommt jedoch hinzu, dass alles funktionierte. Ich habe dann meine Synology und meinen RaspberryMatic komplett neu aufgesetzt und bekomme nun die Geräte, die ich auf Werkseinstellungen zurückgesetzt habe, nicht angelernt.

Nun wollte ich alles von Anfang an richtig schick und ordentlich machen, nun bekomm ich nichts mehr verbunden :(
